src/static/js/modules/util/breakpoint-state.js

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var _breakpointsConfig = require( '../../config/breakpoints-config' );
+var _getViewportDimensions = require( './get-viewport-dimensions' )
+                             .getViewportDimensions;
+
+function _inBreakpointRange( breakpointRange, width ) {
+  var min = breakpointRange.min || 0;
+  var max = breakpointRange.max || Number.POSITIVE_INFINITY;
+
+  return min <= width && width <= max;
+}
+
+/**
+ * @returns {object} An object literal with Boolean
+ * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ * @param {integer} width Current window width.
+ */
+function get( width ) {
+  var breakpointState = {};
+  var breakpointKey;
+  width = width || _getViewportDimensions().width;
+
+  for ( var rangeKey in _breakpointsConfig ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
```

Comment:
Is `guard-for-in` ignore necessary? It probably wants `if ( _breakpointsConfig.hasOwnProperty( rangeKey ) {` to be added in the loop? Or ``if (  _breakpointsConfig.hasOwnProperty( rangeKey ) { continue; }` (see variations [here](http://stackoverflow.com/questions/3765723/is-an-if-with-a-continue-a-good-pattern-to-prevent-excessive-nesting-while-itera))